tests/e2e/specs/endorse-owners.spec.js

```diff
@@ -1,21 +1,27 @@
 import { ACCOUNT_3 } from "../utils";
 
 before(() => {
-  cy.importMetamaskAccount("0xc58c1ff75001afdca8cecb61b47f36964febe4188b8f7b26252286ecae5a8879");
+  // Import Account 2 0xcDFAcbb428DD30ddf6d99875dcad04CbEFcd6E60
+  cy.importMetamaskAccount("0xc58c1ff75001afdca8cecb61b47f36964febe4188b8f7b26252286ecae5a8879"); 
   cy.switchMetamaskAccount(1);
 });
 
-describe("Endorse Owner", () => {
-  it("should endorse owner successfully", () => {
+describe("Endorse Owners", () => {
+  it("should endorse owners successfully", () => {
```

Comment:
```suggestion
describe("Endorse Transfer of Ownership/Holdership", () => {
  it("should endorse transfer of both owner and holder successfully", () => {
```

maybe change the filename too? `endorse-owners.spec.js` to `endorse-owner-holder.spec.js`?